tests/sles4sap/sap_suse_cluster_connector.pm

```diff
@@ -55,31 +55,33 @@ sub run {
 
     # Check HA config and list resources
     my @commands = get_var('NW') ? qw(hcc lsr) : qw(hcc);
-    exec_conn_cmd(binary => $binary, cmd => "$_ --sid $instance_sid --ino $instance_id", log_file => $log_file) foreach (@commands);
+    exec_conn_cmd(binary => $binary, cmd => "$_ --sid ${instance_sid} --ino ${instance_id}", log_file => $log_file) foreach (@commands);
 
     # Test Maintenance Mode
     foreach my $mod (1, 0) {
-        exec_conn_cmd(binary => $binary, cmd => "smm --sid $instance_sid --ino $instance_id --mod $mod", log_file => $log_file);
+        exec_conn_cmd(binary => $binary, cmd => "smm --sid ${instance_sid} --ino ${instance_id} --mod ${mod}", log_file => $log_file);
```

Comment:
Again the `{}` in the variables.

Reply:
Curly braced variable is safely enclosed variable.